From 4bd6ce2a4f6c6b6e31a8476a96f76916f113cc1f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <joerg@thalheim.io>
Date: Fri, 10 Jul 2020 08:53:11 +0100
Subject: [PATCH] hardcode plugin interpreters
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jörg Thalheim <joerg@thalheim.io>
---
 plug-ins/python/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plug-ins/python/meson.build b/plug-ins/python/meson.build
index 95532ed..55846ae 100644
--- a/plug-ins/python/meson.build
+++ b/plug-ins/python/meson.build
@@ -43,7 +43,7 @@ if python.found()
       'python=@0@\n'          .format(python.path()),
       'python3=@0@\n'         .format(python.path()),
       '/usr/bin/python=@0@\n' .format(python.path()),
-      ':Python:E::py::python3:'
+      ':Python:E::py::@0@:'.format(python.path())
     ],
     capture: true,
     install: true,
-- 
2.27.0

